client/extensions/woocommerce/state/data-layer/orders/index.js

```diff
@@ -43,3 +52,15 @@ export function apiError( { dispatch }, action, error ) {
 		dispatch( { ...action.failureAction, error } );
 	}
 }
+
+export function requestOrder( { dispatch }, action ) {
+	const { siteId, orderId } = action;
+	action.failureAction = failOrder( siteId, orderId );
+
+	dispatch( request( siteId, action ).get( `orders/${ orderId }` ) );
```

Comment:
Looks like there is one optional query arg on this endpoint, but guessing we will never need it here? http://woocommerce.github.io/woocommerce-rest-api-docs/#retrieve-an-order `dp`

Reply:
That's a strange option. I don't think we need to worry about it, it looks like it defaults to the user setting for the currency, and we should be formatting prices based on currency values on display anyway.